zero_shot_classification/src/classifier.py

```diff
@@ -51,22 +51,32 @@ def __call__(self, request: dict)-> dict:
             dict: classes of the given text
         """
         texts = [self.get_clean_text(text) for text in request["texts"]]
-        labels = request["labels"]
-        hypothesis = request["hypothesis"]
 
-        model_name = request["model_name"]
-        tokenizer_name = request["tokenizer_name"]
+        labels = request["labels"] if "labels" in request.keys() else config.DEFAULT_CANDIDATE_LABELS
```

Comment:
This works but a cleaner way is
request.get("labels", config.DEFAULT_CANDIDATE_LABELS)

Please change for all